packages/calypso-products/src/is-plan.ts

```diff
@@ -0,0 +1,12 @@
+import { PLAN_HOST_BUNDLE, PLAN_FREE } from './constants';
+import { getPlansSlugs } from './main';
+
+type SnakeCaseProduct = { product_slug: string };
```

Comment:
Tiny nit pick- would `SnakeCaseProductSlug` be a more appropriate name? (And similar for camel case)

Reply:
🤔  to me, that would imply that the type refers to a slug, but it actually refers to an object _with_ a slug. Maybe there's a compromise? ... `ObjectWithSnakeCaseSlug` seems too verbose.

Reply:
(it'd be nice to change this to operate only on the slug itself, but that would mean changing everywhere that uses it)

Reply:
Ah- good point. Maybe `HasSnakeCaseProductSlug`? That sort of makes sense grammatically in a type signature, like `product: HasSnakeCaseProductSlug | HasCamelCaseProductSlug`. It is verbose though.

Reply:
I like that!